docs/application/dotnet/guides/nui/nui-components/Notification.md

```diff
@@ -0,0 +1,52 @@
+# Notification
+Notification is a common component that is help to raise a notification window with a content View.
+
+A Notification can be created using property.

```

Comment:
Suggest rephrasing to:

Notification is a common component that helps to pop-up a notification window with a content view.
A Notification can contain text and can be created using property.

Reply:
Done